echopype/commongrid/api.py

```diff
@@ -17,6 +17,36 @@
 )
 
 
+def _set_var_attrs(da, long_name, units, actual_range_round, standard_name=None):
+    """
+    Attach common attributes to DataArray variable.
+
+    Parameters
+    ----------
+    da : xr.DataArray
+        DataArray that will receive attributes
+    long_name : str
+        Variable long_name attribute
+    units : str
+        Variable units attribute
+    actual_range_round : int
+        Number of digits after decimal point for rounding off actual_range
```

Comment:
How about calling `actual_range_round` something like `digits_round` so that it is more obvious what it is?

Reply:
Sure. I'll leave that to you :wink: